packages/types/src/hub.ts

```diff
@@ -16,16 +16,6 @@ import { User } from './user';
  * working in case we have a version conflict.
  */
 export interface Hub {
-  /**
-   * Checks if this hub's version is older than the given version.
-   *
-   * @param version A version number to compare to.
-   * @return True if the given version is newer; otherwise false.
-   *
-   * @hidden
-   */
-  isOlderThan(version: number): boolean;
```

Comment:
It is not clear to me why such function would be in the `Hub` interface since nowhere in the codebase we call `isOlderThan` and only from inside the `Hub class` is where such function is defined, which leads me to believe that was added for testing purpose, but still, don't think it is needed.